src/domain/entities/BatchLogContent.ts

```diff
@@ -0,0 +1,3 @@
+import { MessageType } from "./Log";
+
+export type BatchLogContent = { content: string; messageType: MessageType }[];
```

Comment:
The canonical advice is: use `something[]` only if something has no spaces. Workarounds: Array<...>, or better, create a separate type for the inner type, and then use it `MySingleType[]`